Doppler.ReportingApi/Infrastructure/SummaryRepository.cs

```diff
@@ -76,5 +76,24 @@ public async Task<SubscribersSummary> GetSubscribersSummaryByUserAsync(string us
                 return result;
             }
         }
+
+        public async Task<SystemUsageSummary> GetSystemUsageAsync(string accountName)
+        {
+            using (var connection = await _connectionFactory.GetConnection())
+            {
+                var databaseQuery = @"
+                SELECT
+                    HasCampaignCreated AS HasCampaingsCreated,
+                    HasListCreated AS HasListsCreated,
+                    HasCampaignSent AS HasCampaingsSent
+                FROM dbo.[User]
+                WHERE Email = @accountName";
```

Comment:
Para no incluir espacios redundantes en el query:

```suggestion
var databaseQuery = @"
SELECT
    HasCampaignCreated AS HasCampaingsCreated,
    HasListCreated AS HasListsCreated,
    HasCampaignSent AS HasCampaingsSent
FROM dbo.[User]
WHERE Email = @accountName";
```

Reply:
y que el query quede fuera de la linea imaginaria de las llaves? 😢 🧨

Reply:
Jeje, si, porque en realidad esta en universo diferente y la línea imaginaria no aplica, fijate que el propio Visual Studio te pone el cursor junto al margen.

En un futuro, tal vez tengas esto: https://github.com/dotnet/csharplang/blob/main/proposals/raw-string-literal.md

Y ahí ambos universos se van a unir.